src/frame_compare/services/report/assets/viewer.css

```
@@ -0,0 +1,278 @@
+    --accent-hover: #7ab8f0;
+    --accent-muted: rgba(91, 164, 230, 0.15);
+    --text-primary: #f0f2f5;
+    --text-secondary: rgba(240, 242, 245, 0.7);
+    --text-muted: rgba(240, 242, 245, 0.5);
+    --border: rgba(255, 255, 255, 0.08);
+    --border-active: rgba(91, 164, 230, 0.5);
+    --danger: #e55353;
+    --success: #4caf50;
+    --divider: #5ba4e6;
+
+    --font-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)